editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -7370,6 +7370,54 @@ void Node3DEditor::_preview_settings_changed() {
 	}
 }
 
+void Node3DEditor::_picker_settings() {
+	// get default color picker mode from editor settings
+	int default_color_mode = EDITOR_GET("interface/inspector/default_color_picker_mode");
+	int picker_shape = EDITOR_GET("interface/inspector/default_color_picker_shape");
+
+	{ //sun color picker settings
+		//reset color modes
+		sun_color->get_picker()->set_hsv_mode(false);
+		sun_color->get_picker()->set_raw_mode(false);
+
+		if (default_color_mode == 1) {
+			sun_color->get_picker()->set_hsv_mode(true);
+		} else if (default_color_mode == 2) {
+			sun_color->get_picker()->set_raw_mode(true);
+		}
+
+		sun_color->get_picker()->set_picker_shape((ColorPicker::PickerShapeType)picker_shape);
+	}
```

Comment:
Using a sub-scope is not necessary. There aren't any local variables.

Reply:
It's been a while, so this means that the outer-most braces in portions like this aren't necessary right?

Reply:
Yes, they don't really serve any purpose.